RecoTracker/PixelSeeding/plugins/alpaka/BrokenLineFit.dev.cc

```diff
@@ -159,11 +159,13 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         }
         brokenline::fastFit(acc, hits, fast_fit);
 
+#ifdef BROKENLINE_DEBUG
         // no NaN here....
```

Comment:
Could you update the comment as well ?
```suggestion
        // any NaN value should cause the track to be rejected at a later stage
```

Reply:
implemented in the last push